README.md

```diff
@@ -62,6 +62,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 
 [<img src="https://user-images.githubusercontent.com/60287642/76142880-bb8fab00-6026-11ea-8522-81fb15f9ff43.gif" width=240>]
 
+ - [Breathing Rotating Animation](https://dartpad.dev/14e84dd83f14c9df47971761463bb474) by [Omi Wakode](https://github.com/OmiWakode)
```

Comment:
Please add a date as well.

Reply:
Also, you need to add it at the end of the list. Please go through the contribution guide before sending a PR.

Reply:
Sorry for the ill formated commit! I've made the changes you asked.

Reply:
Sorry for the ill formatted commit. I've made the changes you asked